lib/apiservers/engine/backends/container.go

```diff
@@ -69,6 +70,13 @@ const (
 	MinCPUs = 1
 	// DefaultCPUs - the default number of container VM CPUs
 	DefaultCPUs = 2
+
+	// Random numbers generated by fair dice roll :)
+	DefaultInitialInterval     = 2 * time.Second
```

Comment:
using a very small initial number and a big multiplier to start fast and then slow down